index.htm

```diff
@@ -13,7 +13,14 @@
     <script src="resources/d3.v3.min.js"></script>
     <script src="resources/rickshaw.min.js"></script>
 
-    <script src="logs/hyper.js"></script>
+    <script type="text/javascript">
+        window.ALL_THE_DATA = [];
+        window.logwer = function(hash, time, type, wer) {
+            window.ALL_THE_DATA.push({ hash: hash, time: time, type: type, wer: wer });
+        };
+    </script>
+
+    <script src="werlog.js"></script>
```

Comment:
Changing from logs/hyper.js will require automation changes: `util/automation.py` defines `populate_previous_logs()` and `save_logs()` to take care of copying content of `logs/` and put that back.